returnn/search.py

```diff
@@ -449,7 +449,7 @@ def run(self):
         d = eval(util.uopen(self.recog_words_file.get_path(), "rt").read())
         assert isinstance(d, dict), "only search output file with dict format is supported"
         with util.uopen(self.out_ctm_file.get_path(), "wt") as out:
-            out.write(";; <name> <track> <start> <duration> <word> <confidence> [<n-best>]\n")
+            out.write(";; <name> <track> <start> <duration> <word> <confidence>\n")
```

Comment:
Can you add a comment explaining that the number of columns matter here, and for what (maybe also giving a ref to the PR here)?